src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -337,10 +375,16 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
     assetPropertyService.assetPropertyNames(lang)
   }
 
+  object TierekisteriInternalServerError {
+    def apply(body: Any = Unit, headers: Map[String, String] = Map.empty, reason: String = "") =
+      ActionResult(ResponseStatus(555, reason), body, headers)

```

Comment:
Should we rather use standard error code, such as 502?
"502 Bad Gateway - The server was acting as a gateway or proxy and received an invalid response from the upstream server."